packages/agoric-cli/registry.yaml

```diff
@@ -0,0 +1,32 @@
+version: 1
+
+chains:
+  ollinet:
```

Comment:
should 'ollinet' become part of our external API?

Reply:
This `registry.yaml` is just a sample, but it needs to be named `registry.yaml` or else `agoric ibc-setup` cannot use it.